src/test/java/com/spatineo/anonymisator/FullStackTest.java

```diff
@@ -87,4 +87,52 @@ public void testRowWithPortsAndCommas() throws Exception
 		assertEquals("{!1{127.0.0.0/24}}:59913, {!1{192.168.1.0/24}}:59913,{!1{127.0.0.0/24}} {!1{192.168.1.0/24}} - - [01/Feb/2016:21:05:40 +0000] ...", result);
 	}
 
+	@Test
+	public void testRowWithNumberLooksLikeIPBut4Digits() throws Exception
+	{
+		DnsLookupResult localhostAddress = new DnsLookupResult();
+		localhostAddress.setSuccess(true);
+		localhostAddress.setReverseName("localhost"); // Note: localhost is converted internally into null (as it's a private address)
+
+		System.out.println(processor);
+		String str ="[08/Nov/2018:14:59:34 +0200] Version number SpatialWeb 8.15.3497.0 ...";
+
+		StringWriter out = new StringWriter();
+		StringReader in = new StringReader(str);
+		processor.process(in, out);
+
+		String result = out.toString();
+		if (result.endsWith("\n")) {
+			result = result.substring(0, result.length()-1);
+		}
+		System.out.println(result);
+
+		assertEquals("[08/Nov/2018:14:59:34 +0200] Version number SpatialWeb 8.15.3497.0 ...", result);
+	}
+
+	@Test
+	public void testRowWithNumberLooksLikeIPButNotIP() throws Exception
```

Comment:
I would add a comment to this test to justify why expected results are sub-optimal. Something like: "The anonymiser cannot tell whether the string 8.15.1.0 is an IP address or not and thus the expected result here is that the version number is anonymised".